### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by sgrossberndt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,44 +20,59 @@ The main services to request information are:
...
 
 ## Standard document
 OJP is a CEN standard. Obtain a copy of the standard document
 ["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7)
 at [CEN](https://www.cen.eu).
 
-OJP is part of the [Transmodel](https://www.transmodel-cen.eu/)) family of standards together with [NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/)  and OPRA. It also contains elements from [DATEX II](https://datex2.eu/).
+OJP is part of the [Transmodel](https://www.transmodel-cen.eu/) family of standards together with
+[NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/) and
+[OpRa](https://www.opra-cen.eu/). It also contains elements from [DATEX II](https://datex2.eu/).
```
Comment (skinkie) [inline README.md:33]: I am not a fan of what happened here. Keep it a single line.

File: README.md [hunk 2]
```diff
@@ -20,44 +20,59 @@ The main services to request information are:
...
-* the enquirers start and end points of a journey along with the specific time and date of departure or arrival, or both;
-* possible travel options along with the specific time and date of departure or arrival, or both, including any possible connections;
+The "[ITS Directive Delegated Regulation for provision of EU-wide multimodal travel information services](https://transport.ec.europa.eu/transport-themes/smart-mobility/road/its-directive-and-action-plan/multimodal-travel-information_en)"
+is the legal framework for travel data access and distributed journey planning in Europe. This initiative
+will provide the necessary requirements to make EU-wide multimodal travel information services accurate and
+available across borders. One of the key requirements concerns linking travel information services for
+distributed journey planning. Upon request, travel information service providers shall provide to another
+information service provider "routing results" based on static, and where possible, dynamic information.
```
Comment (skinkie) [inline README.md:67]: Same as the above, keep it a single line.

File: README.md (hunk 1 above)
Comment (skinkie) [reply README.md:33]: The point is that every time someone needs to change the line, we are mixing formatting with code. Keep these thing a single line, and let your text editor do the wrapping, if that is required.

Comment (sgrossberndt) [reply README.md]: I disagree, but it's not important enough to me that it should block this pull request.
https://about.gitlab.com/blog/2016/10/11/wrapping-text/

File: README.md [hunk 3]
```diff
@@ -20,41 +20,45 @@ The main services to request information are:
...
 * [OJP 1.0](https://github.com/VDVde/OJP/releases/tag/v1.0): The version described in CEN/TS 17118:2017
-* [OJP 1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bug fix release
- * OJP 2.0 in preparation. See the following [branch](https://github.com/VDVde/OJP/tree/changes_for_v1.1) .
+* [OJP 1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bugfix release
+* OJP 2.0 is being prepared in [the develop-branch](https://github.com/VDVde/OJP/tree/develop)
 
-## Github Repository
+## GitHub Repository
+Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP
+and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
```
Comment (skinkie) [inline README.md:40]:
```suggestion
Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
```

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Pull request merged**